src/components/v2/Layout/PageContainer/PageContainerStyles.ts

```diff
@@ -1,11 +1,21 @@
-import { layoutOffset } from '../../../../theme/MuiThemeProvider/muiTheme';
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
 
-export const styles = {
-  main: {
-    flexGrow: 1,
-    p: 3,
-    pb: 6,
-    minHeight: '100vh',
-  },
-  footer: { position: 'fixed', bottom: 0, left: 0, ...layoutOffset },
+export const useStyles = () => {
+  const theme = useTheme();
+  return {
+    main: css`
+      flex-grow: 1;
+      padding: 3;
+      padding-bottom: 6;
```

Comment:
I know it's not your code, but since it's `v2` I'll still review it:
```suggestion
      padding: 3px 3px 6px 3px;
```